src/shared_utilities/drivers.jl

```diff
@@ -1082,6 +1083,46 @@ function perturbed_specific_humidity_from_dewpoint(
     return q
 end
 
+"""
+    perturbed_specific_humidity_from_dewpoint_and_rh(dewpoint_temperature, temperature, air_pressure, earth_param_set, Δrh)
+
+Estimates the perturbed specific humidity given the true dewpoint temperature, true temperature of the air
+in Kelvin, and true air pressure in Pa, along with a perturbation to the relative humidity Δrh, and the ClimaLand
+`earth_param_set`. We first compute relative humidity, and then perturb it to rh + Δrh, compute a perturbed `q`.
+
+Relative humidity is computed using the Magnus formula.
+
+For more information on the Magnus formula, see e.g.
+Lawrence, Mark G. (1 February 2005).
+"The Relationship between Relative Humidity and the Dewpoint Temperature in Moist Air:
+A Simple Conversion and Applications".
+Bulletin of the American Meteorological Society. 86 (2): 225–234.
+"""
+function perturbed_specific_humidity_from_dewpoint_and_rh(
```

Comment:
```suggestion
function perturbed_specific_humidity(
```
I think this is more concise and still readable, but I'm ok either way!

Reply:
I did:
```
ClimaLand.perturbed_rh_specific_humidity_from_dewpoint
ClimaLand.perturbed_temp_specific_humidity_from_dewpoint
```